src/main/java/com/github/kaktushose/jda/commands/internal/CommandDispatcher.java

```diff
@@ -55,7 +56,7 @@ public CommandDispatcher(Object jda,
         this.eventParser = eventParser;
         this.commandMapper = commandMapper;
         this.argumentParser = argumentParser;
-        this.embedFactory = embedFactory;
+        this.embedFactory = new EmbedFactory();
```

Comment:
This means that own embed factories cannot be used. This should be undo

Reply:
For whatever reason embedFactory is null when using `this.embedFactory = embedFactory;``.
I fixed it by just creating a new one.

Reply:
Well but your fix disables other functionality. However the NPE is related to a bug/missing statement I've just found. In the JDACommandsBuilder constructor using the ShardManager, the EmbedFactory is not instantiated, resulting in a NPE

Reply:
Do you want me to correct it, or do you want to do it yourself?